bugbear.py

```diff
@@ -751,7 +753,7 @@ def is_str_or_ellipsis(node):
                     B027(stmt.lineno, stmt.col_offset, vars=(stmt.name,))
                 )
 
-        if not has_abstract_method:
+        if not has_abstract_method and has_method:
```

Comment:
tiny nit: this code is entirely correct, but writing `if has_method and not has_abstract_method:` makes it *obvious* that there's no operator precedence issue, and I find it better matches natural-language grammar.

Not asking for a change, just an FYI.

Reply:
haha, that's a great nit. No reason not to change it~